lib/Widget/evaluationForms/previsitForm/healthStatus.dart

```diff
@@ -0,0 +1,2711 @@
+                                          '_giValue2': _giValue2,
+                                          '_giValue3': _giValue3,
+                                          '_giValue4': _giValue4,
+                                          '_giValue5': _giValue5,
+                                          '_giValue6': _giValue6,
+                                          '_giValue7': _giValue7,
+                                          'giOtherDetail': _gi,
+                                          'msValue1': _msValue1,
+                                          'msValue2': _msValue2,
+                                          'msValue3': _msValue3,
+                                          'msValue4': _msValue4,
+                                          'msValue5': _msValue5,
+                                          'msValue6': _msValue6,
+                                          'msValue7': _msValue7,
+                                          'msOtherDetail': _ms,
+                                          'endocrineValue1': _endocrineValue1,
+                                          'endocrineValue2': _endocrineValue2,
+                                          'endocrineValue3': _endocrineValue3,
+                                          'endocrineValue4': _endocrineValue4,
+                                          'endocrineValue5': _endocrineValue5,
+                                          'endocrineValue6': _endocrineValue6,
+                                          'endocrineValue7': _endocrineValue7,
+                                          'endocrineOtherDetail': _endocrine,
+                                          'lymphValue1': _lymphValue1,
+                                          'lymphValue2': _lymphValue2,
+                                          'lymphValue3': _lymphValue3,
+                                          'lymphValue4': _lymphValue4,
+                                          'lymphValue5': _lymphValue5,
+                                          'lymphValue6': _lymphValue6,
+                                          'lymphValue7': _lymphValue7,
+                                          'lymphOtherDetail': _lymph,
+                                          'abnormalDetails': _abnormalDetails
+                                        };
+
+                                        //**Please help create add form to db function**
+                                        // _firebaseService.addDataToFormsCollection(
+                                        // formName: 'Pre-visit', data: widget.generalForm);
+                                        //               _firebaseService.addDataToFormsCollection(
+                                        // formName: 'Pre-visit', data: widget.adlForm);
+                                        // _firebaseService.addDataToFormsCollection(
+                                        // formName: 'Pre-visit', data: healthStatusForm);
```

Comment:
Please help create add form to db function

Reply:
@prawproudd  Done